src/controllers/blocks/BlocksController.ts

```diff
@@ -169,4 +166,77 @@ export default class BlocksController extends AbstractController<BlocksService>
 			await this.service.fetchBlock(hash, options)
 		);
 	};
+
+	/**
+	 * Return the Header of a block when a block number is queried
+	 *
+	 * @param req Express Request
+	 * @param res Express Response
+	 */
+	private getBlockByIdSummary: RequestHandler<INumberParam> = async (
+		{ params: { number } },
+		res
+	): Promise<void> => {
+		const hash = await this.getHashForBlock(number);
+
+		BlocksController.sanitizedSend(
+			res,
+			await this.service.fetchBlockSummary(hash)
+		);
+	};
+
+	/**
+	 * Return the header of the latest block
+	 *
+	 * @param req Express Request
+	 * @param res Express Response
+	 */
+	private getLatestBlockSummary: RequestHandler = async (
+		{ query: { finalized } },
+		res
+	): Promise<void> => {
+		const paramFinalized = finalized === 'true' ? true : false;
```

Comment:
```suggestion
		const paramFinalized = finalized !== 'false';
```
If we want the default behavior to get the finalized head I think we should do this: if finalized exactly equals false, we don't get the non-finalized head; otherwise get the finalized head.

Reply:
great catch. Resolved.